decompiler/pipeline/controlflowanalysis/variable_name_generation.py

```diff
@@ -128,12 +137,22 @@ def _hungarian_prefix(self, var_type: Type) -> str:
                 return "b"
             elif var_type.size == 0:
                 return "v"
+            else:
+                return ""
```

Comment:
```suggestion
        return ""
```
Or we could either fix the typing of the function or add an assertion/ValueError